source/index.html.md

```diff
@@ -476,7 +476,15 @@ echo $response->getBody();
           "name": "Entertainment"
         }
       ],
-      "has_video": true
+      "has_video": true,
+	  "entities": [
```

Comment:
Style: should this be aligned in the same way as `"topics"`?

Reply:
It is aligned when on the actual site. It just looks odd there as github treats spaces vs tabs differently. I can change it if you want

Reply:
Nope, no need. If it looks right on the site, that's all that matters. I'll change to approved :)